app/views/posts/_comments.html.erb

```diff
@@ -1,6 +1,34 @@
 <% comments.each do |comment| %>
-<div class="comment">
-  <%= link_to comment.account.full_name, profile_path(comment.account.username) %>
-  <%= comment.message %>  
-</div>
-<% end %>
+ <div class = "comment" >
+   <% if comment.account.profile_image.attached? %>
+        <%= link_to image_tag(comment.account.profile_image, width: "30px" , height:"30px", :class => "small-pic float-left m-r-15"),profile_path(comment.account) %>
+    <% else %>
+        <img class="small-pic float-left m-r-15" width="30px" height="30px" src="https://www.iconpacks.net/icons/2/free-reddit-logo-icon-2436-thumb.png">
+    <% end %> 
+  <strong><%= link_to comment.account.full_name, profile_path(comment.account.username), class: "text-dark" %></strong>
+  <%= comment.message %>
+  <div class = "fl">
+  <p class = "text-muted m-l-30"><%= time_ago_in_words comment.created_at %> ago </p>
+   <% if !(@post.closed?) %>
+    <a href = "#" class="comment-form-display">Reply</a>
+   <% end %>
+  <div class = "comment-form">
+     <%= render partial: "comments/form", locals: {post: comment.post, parent: comment, comment_id: comment.id}%>
+  </div>  
+  <hr>
+   <% if comment.replies.any? %>
+     <% comment.replies.each do |reply| %> 
+      <div class = "sub-comment"><%= render partial: 'comments/reply', locals: { reply: reply, comment_id: reply.id } %></div>
+     <% end %>
+   <% end %>
+ </div> 
+<% end %> 
+
+<script>
+  document.querySelectorAll('.comment-form-display').forEach((el) => {
+    el.addEventListener('click', (ev) => {
+     ev.preventDefault();
+     el.nextElementSibling.style = 'display: block;'
+    })
+  })
+</script>
```

Comment:
Please move scripts to a separate js file. Don't include js code like this in html.erb.

Reply:
js files Moved to separate file